### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,16 +14,26 @@ Localization abstractions and implementations for ASP.NET Core applications.
...
 
-### Providers
+### Localization Providers
 
 Community projects adapt _RequestCultureProvider_ for determining the culture information of an `HttpRequest`.
 
- * [My.AspNetCore.Localization.Session](https://github.com/hishamco/My.AspNetCore.Localization.Session) - determines the culture information for a request via values in the session state.
\ No newline at end of file
+ * [My.AspNetCore.Localization.Json](https://github.com/hishamco/My.AspNetCore.Localization.Json) - determines the culture information for a request from a JSON file.
+
```
Comment (Eilon) [inline README.md]: Can you fix the formatting here? Remove the empty line, and fix the indentation of the list items to match.

File: README.md [hunk 2]
```diff
@@ -14,16 +14,26 @@ Localization abstractions and implementations for ASP.NET Core applications.
...
+  * [My.AspNetCore.Localization.Session](https://github.com/hishamco/My.AspNetCore.Localization.Session) - determines the culture information for a request via values in the session state.
+
+  ### Localization Resources
+
+Community projects adapt _IStringLocalizer_ for fetching the localiztion resources.
+
+ * [My.Extensions.Localization.Json](https://github.com/hishamco/My.Extensions.Localization.Json) - fetches the localization resources from JSON file(s).
+
```
Comment (Eilon) [inline README.md]: Same as above.

**Comment by hishamco** [comment]
Any updates on this?

**Comment by Eilon** [comment]
There ya go, sorry for the delay, not sure how I missed this. Maybe because the branch was force-pushed (instead of adding a commit), GitHub doesn't send anotification?

**Comment by hishamco** [comment]
No problem @Eilon, one more thing is there any updates in other PRs

**Comment by Eilon** [comment]
@hishamco I'll make sure we take a look at the open PRs.

**Comment by hishamco** [comment]
Thanks a lot

**Pull request merged**